homeassistant/components/screenlogic/binary_sensor.py

```diff
@@ -54,6 +54,7 @@ async def async_setup_entry(
                 for chem_alarm in coordinator.data[SL_DATA.KEY_CHEMISTRY][
                     SL_DATA.KEY_ALERTS
                 ]
+                if chem_alarm != "_raw"
```

Comment:
Maybe consider excluding anything that starts with `_` for convention in the future?